sdk/core/Azure.Core/README.md

```diff
@@ -196,4 +203,24 @@ KeyVaultSecret secret = client.GetSecret("Name");
 
 More on mocking in [mocking samples](samples/Mocking.md)
 
+## Troubleshooting
+
+Three main ways of troubleshooting failures are [inspecting exceptions](samples/Response.md#Enabling exceptions), enabling [logging](samples/Diagnostics.md#Logging), and [distributed tracing](samples/Diagnostics.md#Distributed-tracing)
+
+## Next steps
+
+Explore and install [available Azure SDK libraries](https://azure.github.io/azure-sdk/releases/latest/dotnet.html).
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][code_of_conduct]. For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments.
+
 ![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Fcore%2FAzure.Core%2FREADME.png)
+
+[source]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/core/Azure.Core/src
```

Comment:
Please have a look at the updated README template at https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md. In particular the list of links have changed to a list and not a single line and the lines to the source tree should be relative instead of absolute links into master.